src/c2goto/library/io.c

```diff
@@ -37,6 +37,16 @@ __ESBMC_HIDE:;
   return ret;
 }
 
+char *gets(char *buf)
+{
+__ESBMC_HIDE:;
+  size_t i = 0;
+  size_t size = nondet_uint();
+  for(i = 0; i < size; i++)
+    buf[i] = nondet_char();
+  return buf;
```

Comment:
The last element should also be a null character: https://www.cplusplus.com/reference/cstdio/gets/

Reply:
Fixed.